From: SeongJae Park <sj@kernel.org>
Date: Sat, 23 Nov 2024 07:29:53 -0800
Subject: [PATCH] samples/damon/prcl: fix type of max_age_region

The type of ->max_age_region of struct damos_access_pattern is unsigned
int, but the sample program is setting it as the max value of unsigned
long type.  Fix it to use the max value of unsigned int type.

Fixes: 6a0ed0a7aac3 ("samples/damon/prcl: implement schemes setup")
Reported-by: kernel test robot <lkp@intel.com>
Closes: https://lore.kernel.org/oe-kbuild-all/202411231420.di7hRYuZ-lkp@intel.com/
Signed-off-by: SeongJae Park <sj@kernel.org>
---
 samples/damon/prcl.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/samples/damon/prcl.c b/samples/damon/prcl.c
index 5f91046d2156..62fbb254ada0 100644
--- a/samples/damon/prcl.c
+++ b/samples/damon/prcl.c
@@ -83,7 +83,7 @@ static int damon_sample_prcl_start(void)
 			.min_nr_accesses = 0,
 			.max_nr_accesses = 0,
 			.min_age_region = 50,
-			.max_age_region = ULONG_MAX},
+			.max_age_region = UINT_MAX},
 			DAMOS_PAGEOUT,
 			0,
 			&(struct damos_quota){},
-- 
2.39.5

